src/components/collection_voting_slab/DataUseAlertBox.js

```diff
@@ -28,8 +28,8 @@ const styles = {
 const dataUseDescriptions = (translatedDataUse) => {
   return flatMap(key => {
     const dataUses = translatedDataUse[key];
-    return map(dataUse => {
-      return div({key: dataUse.code}, [dataUse.description]);
+    return map.convert({cap: false})((dataUse, index) => {
+      return div({key: dataUse.code + '-' + index}, [dataUse.description]);
     })(manuallyReviewedDataUses(dataUses));
```

Comment:
In this new world, we can have duplicate keys with slightly different data uses so we need to ensure key uniqueness here